MyCards/MyCards/CardMO+CoreDataClass.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  CardMO+CoreDataClass.swift
+//  MyCards
+//
+//  Created by Maciej Piotrowski on 07/12/16.
+//  Copyright © 2016 Maciej Piotrowski. All rights reserved.
+//  This file was automatically generated and should not be edited.
+//
+
+import Foundation
+import CoreData
+import UIKit
+
+public class CardMO: NSManagedObject {}
+
+extension CardMO: ManagedObjectProtocol {
+    static var defaultSortDescriptors: [NSSortDescriptor] {
+        return [NSSortDescriptor(key: "name", ascending: true)]
+    }
+
+    func toEntity() -> Card? {
+        var front: UIImage?
+        var back: UIImage?
+        self.front.map { front = UIImage(data: $0 as Data) }
```

Comment:
images kept as data in Core Data - not efficient. 
Is save to disk flag set for these properties?
Consider saving to filesystem and store url in CD